src/annotator/plugin/cross-frame.coffee

```diff
@@ -64,8 +65,14 @@ module.exports = class CrossFrame extends Plugin
         # Take the embed script location from the config
         # until an alternative solution comes around.
         embedScriptUrl = config.embedScriptUrl
+
         FrameUtil.isLoaded frame, () ->
-          FrameUtil.injectHypothesis(frame, embedScriptUrl, config)
+          subFrameIdentifier = discovery._generateToken()
```

Comment:
I see we have an integration test for multi-frame support, which is great, but not currently unit tests for this behaviour in `cross-frame-test.coffee`.

Ideally we should have unit tests for all changes and integration tests for important behaviour that covers a lot of modules (like this multi-frame support).